ci/nightly/pipeline.template.yml

```diff
@@ -1302,6 +1302,17 @@ steps:
         agents:
           queue: linux-x86_64-small
 
+  - id: qa-canary-load
```

Comment:
This is going to run for all the PRs as well , do we want that?

Reply:
No, good catch.